app/addons/documents/routes-index-editor.js

```diff
@@ -0,0 +1,155 @@
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+
+  // Modules
+  'addons/documents/shared-routes',
+  'addons/documents/views',
+  'addons/documents/views-index',
+  'addons/databases/base',
+  'addons/fauxton/components'
+
+],
+
+function (app, FauxtonAPI, BaseRoute, Documents, Index, Databases, Components) {
+
+
+  var IndexEditorAndResults = BaseRoute.extend({
+    layout: 'two_pane',
+    routes: {
+      'database/:database/new_view': 'newViewEditor',
+      'database/:database/new_view/:designDoc': 'newViewEditor',
+      'database/:database/_design/:ddoc/_view/:view': {
+        route: 'viewFn',
+        roles: ['fx_loggedIn']
+      }
+    },
```

Comment:
most of it is copy pasted from routes.js  - main difference is the usage of breadcrumbs and omitted header components